### DISCUSSION TITLE
Is there a way to analyze multiple images at the same time?

### DISCUSSION THREAD

**Discussion by mrek235** [PolarizedLightFieldMicroscopy/napari-LF#51] [Ideas]
Hello,

I wanted to ask if there is a way to mass analyze images using the Napari gui. In our lab we usually have several thousand images per experiment, and if we could just select a folder -and maybe a list of GPUs to distribute the images over- and then run the calibration and deconvolution, it would be really great for us.

**Comment by gschlafly** [comment]
Hi @mrek235! I think we can figure out a way to make that possible. Within each experiment, are all of your desired calibration settings the same across your images? If so, then we can make a workflow where the calibration process is run once and then the deconvolution process is run on every image in a provided folder.

**Comment by mrek235** [comment]
Hello! Yes, all the images are deconvolved using the same calibration
settings. We normally use LFAnalyze 0.4's lfcalibrate to create the
calibration file by calibrating on an averaged image from the experiment. I
think that kind of a workflow would be really awesome and helpful.
